docs/source/video_tutorials.rst

```diff
@@ -0,0 +1,39 @@
+Video Tutorials
+===============
+
+Basics of *stk*
+---------------
+
+https://www.youtube.com/watch?v=mPr9D7nCQ84&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=1
+
+Functional Groups in *stk*
+--------------------------
+
+https://www.youtube.com/watch?v=IaLIN7jXyKQ&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=2
+
+Cage Building
+-------------
+
+https://www.youtube.com/watch?v=sc1Kf6vzTSA&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=3
+
+
+Hierarchical Construction with *stk*
+------------------------------------
+
+https://www.youtube.com/watch?v=WB9oAqA8m4o&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=4
+
+Using *stko* with *stk*
+-----------------------
+
+https://www.youtube.com/watch?v=vvmS43dVPe4&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=5
+
+
+Databases and *stk-vis* in *stk*
+--------------------------------
```

Comment:
A short introduction to databasing and the use of _stk-vis_ with _stk_ molecules!

Jupyter Notebook is available at https://github.com/andrewtarzia/stk-examples/tree/main/notebooks_from_videos .